### ISSUE TITLE
command:pickRemoteProcess with pipeTransport fails on linux target running procps 3.2.8

### ISSUE THREAD

**Issue by Ezekiel-DA** [microsoft/vscode-cpptools#2183]
Type: Debugger

# Details
- OS and Version: Windows 10 (version 1703)
- VS Code Version: 1.24.1
- C/C++ Extension Version: 0.17.5
- Other extensions you installed (and if the issue persists after disabling them): N/A
- A clear and concise description of what the bug is:
VS Code fails to list the remote processes running on the target remote (found through `pipeTransport` to a docker container, in this case) for specific versions of `ps`.

This error is produced:
![image](https://user-images.githubusercontent.com/10823051/41860709-46568194-786d-11e8-85a6-932f6520d221.png)

This error originates from the underlying call to `ps` through the pipe.

This occurs when the target is running `procps` (version 3.2.8, and probably others). 

As per [this man page for `ps`](https://linux.die.net/man/1/ps), the command issued by the extension (`ps -axww`) is technically an invalid syntax.
`-x` is not an available Unix style (dashed) option; `x` (no dash) is a valid BSD style option. This specific version of `ps` appears to honor the man page's suggestion that a warning should be shown when option styles are accidentally mixed together. Said warning trips up this extension's attempt at parsing the results of `ps`.

This goes unnoticed for other flavors of Linux (incl. CentOS 7) because they appear to use `procps-ng`, which silently honors the option without first issuing a warning. I am however stuck with a need for a series 6 CentOS solution!

FWIW, `ps axww` (no dash) works correctly for both packages (`procps-ng` and regular old `procps`) on all the version I had on hand to test.

# Reproduction steps
If you have Docker for Windows installed, getting the correct version of `ps` to reproduce this is as easy as:

```
docker run -it --name repro centos:6.9
```

This launch.json file can then be used to reproduce the problem by simply attempting to start a debugging session:
```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "C++: Gdb attach in 'repro' container",
            "type": "cppdbg",
            "request": "attach",
            "program": "/bin/bash",
            "processId": "${command:pickRemoteProcess}",
            "pipeTransport": {
                "pipeCwd": "${workspaceRoot}",
                "pipeProgram": "docker",
                "pipeArgs": [
                    "exec",
                    "-i",
                    "repro",
                    "sh",
                    "-c"
                ],
                "debuggerPath": "/usr/bin/gdb"
            },
            "linux": {
                "MIMode": "gdb",
                "setupCommands": [
                    {
                        "description": "Enable pretty-printing for gdb",
                        "text": "-enable-pretty-printing",
                        "ignoreFailures": true
                    }
                ]
            }
        }
    ]
}
```

**Comment by WardenGnaw** [comment]
Workaround would be to delete the `-` in .vscode\extensions\ms-vscode.cpptools-<version>\out\src\Debugger\nativeAttach.js for psLinuxCommand.

**Comment by sean-mcmanus** [comment]
Should be fixed with 0.17.6.
